_stbt/core.py

```diff
@@ -1806,14 +1806,23 @@ def draw(self, img):
         _draw_text(img, self.label, label_loc, (255, 255, 255), font_scale=0.5)
 
 
+class _TextAnnotation(namedtuple("_TextAnnotation", "time text duration")):
+    @property
+    def end_time(self):
+        return self.time + self.duration
+
+
 class SinkPipeline(object):
     def __init__(self, user_sink_pipeline, raise_in_user_thread, save_video=""):
+        import time as _time
+
         self.annotations_lock = threading.Lock()
         self.text_annotations = []
         self.annotations = []
         self._raise_in_user_thread = raise_in_user_thread
         self.received_eos = threading.Event()
         self._frames = deque(maxlen=35)
+        self._time = _time
```

Comment:
I guess you'd originally intended to use this in tests?  Doesn't look necessary any more.